src/utilities/__tests__/buildASTSchema-test.js

```diff
@@ -763,6 +772,39 @@ describe('Schema Builder', () => {
     });
   });
 
+  it('Supports @specifiedBy', () => {
+    const sdl = dedent`
+      scalar Foo @specifiedBy(url: "https://example.com/foo_spec")
+
+      type Query {
+        foo: Foo @deprecated
+      }
+    `;
+    expect(cycleSDL(sdl)).to.equal(sdl);
+
+    const schema = buildSchema(sdl);
+
+    const foo = assertScalarType(schema.getType('Foo'));
+
+    expect(foo).to.include({
+      specifiedByUrl: 'https://example.com/foo_spec',
+    });
+  });
+
+  it('@specifiedBy should not be allowed multiple times', () => {
+    const sdl = dedent`
+      scalar Foo @specifiedBy(url: "https://first.com/foo_spec") @specifiedBy(url: "https://second.com/foo_spec")
+
+      type Query {
+        foo: Foo @deprecated
+      }
+    `;
+
+    expect(() => buildSchema(sdl)).to.throw(
+      'The directive "@specifiedBy" can only be used once at this location.',
+    );
+  });
```

Comment:
No need to test for test this error specifically for `@specifiedBy` since it's a general rule for all directives and already tested inside `src/validation/__tests__/`.